.php_cs

```diff
@@ -0,0 +1,63 @@
+<?php
+
+if (class_exists('Symfony\CS\Config\Config')) {
+    // PHP-CS-Fixer 1.x syntax (deprecated)
```

Comment:
you don't need this, the reason why it is there on kernel was for the transition period between 1.x and 2.x. But we don't support using both, 1.x is by now dead.

Reply:
https://github.com/ezsystems/ezpublish-kernel/commit/b8af033907e7cdb70c388925da66c354e1a67dda